media-sound/jack/jack-4.0.0_pre20200706.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
+PYTHON_REQ_USE="ncurses"
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="A frontend for several cd-rippers and mp3 encoders"
+HOMEPAGE="https://github.com/jack-cli-cd-ripper/jack http://www.home.unix-ag.org/arne/jack/"
+GIT_COMMIT_HASH="8e2004d7453f5d71d5e73032cbe7770b75bfbd41" # branch "python3-mb"
+SRC_URI="https://github.com/jack-cli-cd-ripper/jack/archive/${GIT_COMMIT_HASH}.tar.gz -> ${PN}-${PV}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Might be GPL-2+ based on the README and header of file(s?)

Reply:
Will investigate...

Reply:
yes, GPL-2+